### DISCUSSION TITLE
How to perform inference with EnhS2T model?

### DISCUSSION THREAD

**Discussion by darryllam** [espnet/espnet#5727] [Q&A] [answered]
I'd like to try using the Yoshiki/chime4_enh_asr1_wpd_wavlm_conformer model from Hugging Face, but I do not see any inference functions related to EnhS2T. Is there currently a way to perform inference using Python?
Also are there still plans to add the configurations of https://arxiv.org/abs/2307.12231 to ESPnet?

**Comment by sw005320** [comment, marked as answer]
Thanks for your report.
You can check https://github.com/espnet/espnet/blob/3858d84051d6bed263cefb968bb1727452012cf2/egs2/TEMPLATE/enh_asr1/enh_asr.sh#L1386

- ASR: https://github.com/espnet/espnet/blob/master/espnet2/bin/asr_inference.py

- SE: https://github.com/espnet/espnet/blob/master/espnet2/bin/enh_inference.py
